addon/mixins/sizing-mixin.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @module ticketfly-fields
+ */
+import Ember from 'ember';
+const { Mixin, computed, get } = Ember;
+
+/**
+  @class SizingMixin
+  @extends Ember.Mixin
+ */
+export default Mixin.create({
+  sizing: computed('sizeStyle','utilityName', {
+    get() {
+      const sizeStyle = get(this, 'sizeStyle');
+      if (sizeStyle) {
+        const utilityName = get(this, 'utilityName');
+        return `u-${utilityName}-${sizeStyle}`;
```

Comment:
Is it:
`ticketfly-fields__input--large`
`ticketfly-fields__label--large`
or
`ticketfly-input--large`
`ticketfly-label--large`